app/javascript/components/applications/application_form/preferences/PreferenceForm.js

```diff
@@ -32,12 +32,14 @@ const findPreferencesNotSelected = (formApi, listingPreferences, selectedPrefere
 }
 
 const buildListingPreferencesOptions = (preferencesNotSelected) => {
-  return map(preferencesNotSelected, (listingPref) => {
+  const listingPrefOptions = map(preferencesNotSelected, (listingPref) => {
     return {
       value: listingPref.id,
-      label: listingPref.lottery_preference.name
+      label: listingPref.lottery_preference.name,
+      order: listingPref.order
```

Comment:
added order here so that the preferences are sorted in a consistent order, that matches the listing preference order. It doesn't affect the saving of preferences at all.